src/conversions.jl

```diff
@@ -126,3 +126,6 @@ coloralpha(c::C) where {C<:Color} = coloralpha(C)(c)
 coloralpha(c::C,a) where {C<:Color} = coloralpha(C)(c,a)
 coloralpha(c::C) where {C<:TransparentColor} = coloralpha(base_color_type(C))(color(c), alpha(c))
 coloralpha(c::C,a) where {C<:TransparentColor} = coloralpha(base_color_type(C))(color(c), a)
+
+# Tuple
+Tuple(c::Colorant{T, N}) where {T, N} = (comps(c)...,)::NTuple{N, T}
```

Comment:
Is this the recommended API that people should use?

Reply:
Strictly speaking, I don't think the `Tuple` constructors are a part of public APIs because they don't have docstrings.
However, I don't see a problem with ColorVectorSpace.jl etc. using it internally.

Reply:
Of course, some people might argue that `Colorant` is never iterable. However, since the design of `tuple()` and `Tuple()` are clearly different, I don't think there will be any confusion.

Reply:
I think we need to discuss again whether `Tuple` should be overloaded, and if so, what should be returned. (cf. PR #184)
Currently (ColorTypes v0.11) `Tuple(::Colorant)` causes an error.
```julia
julia> Tuple(RGB(1.0, 0.0, 0.0))
ERROR: MethodError: no method matching iterate(::RGB{Float64})
```

Reply:
I'm personally in favor of the `Tuple(::Colorant)` usage. It's kind of like `CartesianIndex`, that you can't iterate it but can easily construct an iterable tuple from `Tuple` or `x.I`.